frontend/kesaseteli/employer/src/__tests__/utils/components/render-component.tsx

```diff
@@ -13,7 +16,10 @@ const renderComponent = (
 ): RenderResult =>
   render(
     <BackendAPIContext.Provider value={AxiosTestContext}>
-      <QueryClientProvider client={client}>{Component}</QueryClientProvider>
+      <QueryClientProvider client={client}>
+        <ThemeProvider theme={theme}>{Component}</ThemeProvider>
+        <HiddenLoadingIndicator />
```

Comment:
Helper indicator for the tests which will tell if some backend request is till pending